cmd/eofdump/eofparser.go

```diff
@@ -138,19 +138,28 @@ func eofParser(ctx *cli.Context) error {
 	}
 
 	// If neither are passed in, read input from stdin.
-	scanner := bufio.NewScanner(os.Stdin)
-	for scanner.Scan() {
-		t := strings.TrimSpace(scanner.Text())
-		if len(t) == 0 || t[0] == '#' {
+	reader := bufio.NewReaderSize(os.Stdin, 1024*1024)
+	t, err := reader.ReadString('\n')
+	for err == nil {
+		l := len(t)
+		if l == 0 || t[0] == '#' {
 			continue
 		}
-		if _, err := parseAndValidate(t, false); err != nil {
+		if t[l-1] == '\n' {
+			t = t[:l-1] // remove newline
+		}
+		_, err := parseAndValidate(t, false)
+		if err != nil {
```

Comment:
This is strange. Because, there's already an `err` from the outer scope, so here you re-define and shadow the outer `err`. On the line(s) below, I'm not really sure which `err` is being referenced. I assume you'd be better off doing `_, err = parseAndValidate(..` instead

Reply:
Fixed upstream